templates/kube-apiserver.yaml.erb

```diff
@@ -15,7 +15,7 @@ spec:
     - --allow-privileged=true
     <% if @kubernetes_version =~ /1[.](8|9)[.]\d/ -%>- --enable-bootstrap-token-auth=true<% end %>
     <% if @kubernetes_version =~ /1[.](6|7)[.]\d/ -%>- --experimental-bootstrap-token-auth=true<% end %>
-    - --service-cluster-ip-range=10.96.0.0/12
```

Comment:
This value here is also incorrect, this should be the service cidr not the cluster cidr